### ISSUE TITLE
Fix S2372: Rule should not raise an issue on properties throwing NotImplementedException

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#151]
The following code raises but should not:
```
public DateTime RevisedDate
{
    get
    {
        throw new NotImplementedException();
    }
}
```

Implements [RSPEC-2372](https://jira.sonarsource.com/browse/RSPEC-2372).

Initial thread: [SONARCS-715](https://jira.sonarsource.com/browse/SONARCS-715)

**Comment by Evangelink** [comment]
Duplicates #191
